.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi**, and look through [issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=) before submitting.
+
+Check out our [wiki](https://github.com/alexa-pi/AlexaPi/wiki/Debugging), and [Audio Debugging](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging) in particular before filing an issue.
+
+In order to help troubleshooting, be sure to include the following information:
+
+**Output of the [audio debugging script](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging#audio-debugging-script).**
+
+**Your OS (including version) where you are running AlexaPi:**
+```
+```
+
+**Your hardware platform and model you are running on:**
+```
+
+```
+
+**Python release (`python2 --version`):**
+```
+
+```
+
+**Description of problem:**
+```
+
+```
+
+**Expected:**
+```
+
+```
+
+**Problem-relevant `config.yaml` entries:**
+```
+
+```
+
+**Steps to reproduce:**
+
+[ ] 1.
```

Comment:
Maybe just 

1.
2.
3.

(only a blank line before the first compared to the previous version - it didn't display as a list before, but all in one line)